app/src/components/Controls.tsx

```diff
@@ -69,14 +69,65 @@ export function IconButton(
     clicked?: boolean;
     active?: boolean;
     icon?: IconType;
+    text?: string;
   } & ButtonHTMLAttributes<HTMLButtonElement>
 ): JSX.Element {
-  const children = props.icon ? <props.icon /> : <></>;
+  const theme = useTheme();
   const onClick = props.active === false ? undefined : props.onClick;
+  const children = props.icon ? <props.icon style={{ color: theme.fg }} /> : <></>;
 
-  return (
+  const iconButton = (
     <IconButtonContainer {...props} onClick={onClick}>
       {children}
     </IconButtonContainer>
   );
+  if ((props.active === undefined || props.active) && props.text) {
+    return (
+      <Popup
+        trigger={() => iconButton}
+        position={['bottom center', 'left center', 'right center']}
+        on={['hover', 'focus']}
+      >
+        <span>{props.text}</span>
+      </Popup>
+    );
```

Comment:
The popup text does not match the `theme.fg`:
<img width="221" alt="Screenshot 2021-10-01 at 19 07 42" src="https://user-images.githubusercontent.com/6545097/135660003-a229fcca-b4f3-445c-a03c-7472e635ac77.png">